### DISCUSSION TITLE
Hoping for Help

### DISCUSSION THREAD

**Discussion by TARGAT321** [Exarkuniv/RetroPie-Extra#192] [Q&A]
hello!  i came across this repo looking for a way to get openbor going on my pi4, running retroarch/emulation station.  i am completely new to any kind of coding/pi set up, so forgive me if i suck at explaining my issues.  before we get to that, i just wanted to say thanks for what i can only imagine is a shit-ton of time put into this.
so i installed everything according to the readme file.  im at a point now where i see openbor in /home/pi/RetroPie/roms/openbor
but now what?  this is where i would normally dump my roms for other systems, and having done that, the system would show in my emulation station carousel.  However, when i put .pak files i have for openbor into the roms folder, it does not show as a system in my carousel.  where should i put these games so they become playable?  i am probably missing something very obvious, but i just don't know enough to know.  any assistance is appreciated.  apologies in advance.

**Comment by Exarkuniv** [comment]
Sorry. I just saw this.

I'll take a look at my build and see what I have going then I can help better
